salt/iiif/config/var-www-loris2-loris2.wsgi

```diff
@@ -1,10 +1,11 @@
-#!/usr/bin/env python
-import site;
-site.addsitedir('/opt/loris/venv/lib/python2.7/site-packages')
-import newrelic.agent
+#!/usr/bin/env python3
+
```

Comment:
this could possibly be baked into the image? Unless the image is pushed upstream and should be generic.

Reply:
definitely a decision to make.

we want insight into the running program like we currently do, so I'll install newrelic within the container and bake it in the configuration.

not sure how it knows where to send the data with no other newrelic configuration going on. I'll investigate that Monday.

Reply:
newrelic APM has been baked in. it can be activated by running the container with `NEW_RELIC_ENABLED=true` as an environment var